docs/docs/validators/partial-set-security-for-validators.md

```diff
@@ -0,0 +1,118 @@
+---
+sidebar_position: 6
+---
+
+# Partial Set Security
```

Comment:
I know it's a bit silly, but maybe consider adding a subsection "How many chains can I opt in to?" because this might be a question that some people have, coming from maybe Eigenlayer where capital is "attributable"